src/model/model.js

```diff
@@ -144,10 +163,11 @@ export default class Model {
 	 *			writer.insertText( 'foo', paragraph, 'end' );
 	 *		} );
 	 *
-	 * @fires event:change
-	 * @fires event:changesDone
+	 * `Batch` instance can be obtained from {@link module:engine/model/writer~Writer#batch the writer}.
+	 *
+	 * @fires event:_change
```

Comment:
I'm not sure if we need to document protected event.

Reply:
Do we need to document protected methods or private methods?

Yeah, that's what I thought :)

Reply:
Documentation to the event is fine. What I don't like is the documentation to the method that it fires event. There is no way to mention that this event is protected and it will be displayed in the public documentation too. This is why I prefer to have no information that this event is fired here.